### PR TITLE
Issue 479 template customization

### PR DESCRIPTION
**Pull request by hallvard**
property.

Thanks for contributing.

- [x] Tick to sign-off your agreement to the [Developer Certificate of Origin (DCO) 1.1](https://developercertificate.org)

## Description

Modified the template to support setting location of plotly js library, and Page and PageBuilder accordingly.

## Testing

There are tests for both cases, the default and custom cases.

### CODE REVIEW

**Comment by hallvard** [comment]
The test succeeds locally when run using mvn clean test. jsplot defines the following dependency on junit:
```
        <dependency>
            <groupId>junit</groupId>
            <artifactId>junit</artifactId>
            <scope>test</scope>
        </dependency>
```

**Comment by benmccann** [comment]
Not anymore: https://github.com/jtablesaw/tablesaw/blob/master/jsplot/pom.xml#L65

We just upgraded to junit 5. Can you rebase against master?

**Comment by hallvard** [comment]
Hm, for some reason I don't get the changes, I'm told my upstream/master branch pointing to https://github.com/jtablesaw/tablesaw.git is up-to-date and the last commit I see is "Use static imports for junit asserts"

**Comment by hallvard** [comment]
Strange, when I turned off VPN I suddenly could fetch the latest changes.

**Comment by hallvard** [comment]
I use Assertions.assertTrue, isn't that good enough?

**Comment by benmccann** [comment]
Perhaps it doesn't have great support for junit 5 yet. Everywhere else we statically import `assertTrue` and I'm guessing that makes it happier. Can you try that?

File: jsplot/src/main/java/tech/tablesaw/plotly/components/Page.java [hunk 1]
```diff
@@ -17,11 +17,18 @@
...
 
+    private String plotlyJsLocation = null;
+
     private Page(PageBuilder builder) {
         this.figure = builder.figure;
         this.divName = builder.divName;
+        this.plotlyJsLocation = builder.plotlyJsLocation;
     }
 
+    public void setPlotlyJsLocation(String plotlyJsLocation) {
```
Comment (benmccann) [inline jsplot/src/main/java/tech/tablesaw/plotly/components/Page.java]: Could we remove this since it's unused and make `plotlyJsLocation` final?

Comment (hallvard) [reply jsplot/src/main/java/tech/tablesaw/plotly/components/Page.java]: Of course, with a builder it is nonsense.

**Comment by hallvard** [comment]
I thought about several designs, one involved a global (singleton) TemplateManager that configured the engine and loader and that had the setTemplateLocations method. But I couldn't come up with other reasons for exposing the engine and loader than this method, so I decided on just introducing the method as a start. We can still keep the method if we need to introduce a (replaceable) singleton later. I haven't tried the method it in Eclipse, yet, but will do it now.

**Comment by hallvard** [comment]
I seemed to work as expected, I could use a different template located in the Eclipse workbench instead of the jsplot jar!

**Pull request merged**